serverless-aws-csharp/${PROJECT}.csproj

```diff
@@ -8,8 +8,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="Pulumi" Version="3.*" />
-		<PackageReference Include="Pulumi.Aws" Version="5.*" />
-		<PackageReference Include="Pulumi.AwsApiGateway" Version="1.*" />
+		<PackageReference Include="Pulumi.Aws" Version="6.*" />
+		<PackageReference Include="Pulumi.AwsApiGateway" Version="1.1.0-alpha.3" />
```

Comment:
Why is a different version specified here for C#, when the other language versions for this architecture specify version 2.x?

Reply:
An oversight, thank you for spotting it!